app/parse.py

```
@@ -15,8 +27,81 @@ class Product:
     num_of_reviews: int
 
 
+def accept_cookies(driver: WebDriver) -> None:
+    try:
+        wait = WebDriverWait(driver, 5)
+        button = wait.until(EC.element_to_be_clickable((By.CLASS_NAME, "acceptCookies")))
+        button.click()
+    except Exception:
+        pass
+
+
+def more_button(driver: WebDriver) -> None:
+    try:
+        while True:
+            products_before = len(driver.find_elements(By.CSS_SELECTOR, ".card.thumbnail"))
+            try:
+                more = driver.find_element(By.CLASS_NAME, "ecomerce-items-scroll-more")
+                if not more.is_displayed():
+                    break
+                more.click()
+            except (NoSuchElementException, ElementNotInteractableException):
+                break
+            WebDriverWait(driver, 5).until(
+                lambda d: len(d.find_elements(By.CSS_SELECTOR, ".card.thumbnail")) > products_before
+            )
+    except Exception:
+        pass
```

The outer `try/except Exception` in `more_button` swallows all errors. Instead, catch expected exceptions (like `TimeoutException`) and allow unexpected exceptions to propagate (or at least log them). That will make silent failures easier to detect.